catalogs/catalog-jdbc-oceanbase/src/main/java/org/apache/gravitino/catalog/oceanbase/operation/OceanBaseTableOperations.java

```diff
@@ -93,18 +274,447 @@ protected String generatePurgeTableSql(String tableName) {
   @Override
   public void alterTable(String databaseName, String tableName, TableChange... changes)
       throws NoSuchTableException {
-    throw new UnsupportedOperationException("Not implemented yet.");
+    LOG.info("Attempting to alter table {} from database {}", tableName, databaseName);
```

Comment:
why need override this method? Can't OBD perform multiple changes in one statement?

Reply:
ok, plz add comment for the code

Reply:
Added.